emr_run_spark.py

```diff
@@ -108,38 +108,13 @@ def _create_steps(job_flow_name=None,
         'Args': ['unzip', zip_file_on_host, '-d', sources_on_host]
       }
     })
-    if use_mysql:
-      steps.append({
-        'Name': 'setup - jars 1',
-        'ActionOnFailure': 'CANCEL_AND_WAIT',
-        'HadoopJarStep': {
-          'Jar': 'command-runner.jar',
-          'Args': 'touch /home/hadoop/noop.py'.split()
-        }
-      })
-      steps.append({
-        'Name': 'setup - jars 2',
-        'ActionOnFailure': 'CANCEL_AND_WAIT',
-        'HadoopJarStep': {
-          'Jar': 'command-runner.jar',
-          'Args': 'spark-submit --packages mysql:mysql-connector-java:5.1.38 /home/hadoop/noop.py'.split()
-        }
-      })
-      steps.append({
-        'Name': 'setup - jars 3',
-        'ActionOnFailure': 'CANCEL_AND_WAIT',
-        'HadoopJarStep': {
-          'Jar': 'command-runner.jar',
-          'Args': 'sudo cp /home/hadoop/.ivy2/jars/mysql_mysql-connector-java-5.1.38.jar /usr/lib/hadoop/'.split()
-        }
-      })
     steps.append({
       'Name': 'run spark {}'.format(spark_main),
       'ActionOnFailure': 'CANCEL_AND_WAIT',
       'HadoopJarStep': {
         'Jar': 'command-runner.jar',
         'Args': ['spark-submit', '--packages',
-                 'mysql:mysql-connector-java:5.1.38', '--py-files',
+                 'mysql:mysql-connector-java:5.1.39', '--py-files',
```

Comment:
LGTM, but my only comment is  that right now `--packages mysql:mysql-connector-java:5.1.39` is added regardless of the flag `use_mysql`. This is incorrect. I realize it was like this before - but please fix it...